### ISSUE TITLE
Localisation metrics in semi-supervised context.

### ISSUE THREAD

**Issue by marc-moreaux** [fchollet/keras#5419]
Hello,

I'm trying to figure how good is the model performing on its localisation abilities.
The localisation is never used for training (it's semi-supervised learning).
Still, I was wondering if there is a way to create a function as in keras.metrics which could be givent the Xs and Yx localisation parameters (Which implies than my batch gives these parameters).

Thank you very much :)

(Another way to think about it is to have many values for every true_y (true_y0, x1, x2, y1, y2) and evaluate the loss only on the true_y0 (1st of the 5 values given as true_y))
